grpcbp/server_middlewares.go

```diff
@@ -164,14 +164,14 @@ func StartSpanFromGRPCContext(ctx context.Context, name string) (context.Context
 //   - grpc_success: "true" if status is OK, "false" otherwise
 //   - grpc_type: type of request, i.e unary
 //   - grpc_code: the human-readable status code, e.g. OK, Internal, etc
-func InjectPrometheusUnaryServerInterceptor(serviceSlug string) grpc.UnaryServerInterceptor {
+func InjectPrometheusUnaryServerInterceptor(serviceName string) grpc.UnaryServerInterceptor {
 	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (_ interface{}, err error) {
 		start := time.Now()
 
 		method := methodSlug(info.FullMethod)
```

Comment:
https://pkg.go.dev/google.golang.org/grpc#UnaryServerInfo

this gives both the serviceName and the method

Reply:
nice. let me fix. thanks.